src/andromede/simulation/xpansion.py

```diff
@@ -0,0 +1,213 @@
+# Copyright (c) 2024, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+
+"""
+The xpansion module extends the optimization module
+with Benders solver related functions
+"""
+
+import json
+import os
+import subprocess
+import sys
+from typing import Any, Dict, List
+
+from andromede.simulation.optimization import (
+    BlockBorderManagement,
+    OptimizationProblem,
+    build_problem,
+)
+from andromede.simulation.time_block import TimeBlock
+from andromede.study.data import DataBase
+from andromede.study.network import Network
+from andromede.utils import serialize
+
+
+class XpansionProblem:
```

Comment:
I agree that we could rename it to something not tool related, but since we can solve a investment problem doing a frontal LP, I don't think this name would be very clear.

The main idea here is to separate both master and sub-problem to use the Benders solver. In that case, BendersProblem would be better, no ?

Reply:
Yes exactly I suggest `BendersProblem` or `DecomposedProblem` or `BendersDecomposedProblem` as we do not want to have any operational knowledge at problem / solver level

Reply:
The idea here is really that we provide a decomposed form of the big initial problem (and more specially a decomposition suited for a resolution with a Benders algorithm)

Reply:
Done. I went for `BendersDecomposedProblem`